src/store/index.js

```diff
@@ -2,17 +2,30 @@ import Vue from 'vue';
 import Vuex from 'vuex';
 
 import mutation from './mutation-types';
+import action from './action-types';
+
+function selectionOffset(state, targetIndex, modeAdd) {
```

Comment:
```suggestion
function offsetSelection(state, targetIndex, modeAdd) {
```
`selectionOffset` ← décalage de la sélection / offset est un nom commun (variable)
`offsetSelection` ← décale  la sélection / offset est un verbe (fonction)